src/cli/lib/printer/printer.ts

```diff
@@ -47,6 +49,19 @@ type Style = {
 	decoration?: Decoration;
 };
 
+const printLoader = () => {
+	const p = ['⠙', '⠘', '⠰', '⠴', '⠤', '⠦', '⠆', '⠃', '⠋', '⠉'];
+	let x = 0;
+	return setInterval(() => {
+		process.stdout.write('\r' + p[x++]);
+		x &= 3;
+	}, 250);
+};
+
+const removeLoader = () => {
+	process.stdout.write('\r\x1b[K');
```

Comment:
Should we also do a `clearInterval()` when removing the loader?

Reply:
Agree. Though this is a CLI tool and the process shouldn't be kept alive too long, it's a good practice to clear the interval to free memory.